scripts/generate/mockgen.go

```diff
@@ -50,29 +50,20 @@ func main() {
 	packageName := os.Args[1]
 	interfaces := os.Args[2]
 	outputPath := os.Args[3]
-	re := regexp.MustCompile("(?m)[\r\n]+^.*" + projectVendor + ".*$")
-
+	re := regexp.MustCompile(projectVendor)
```

Comment:
This is now no longer the same regular expression (or really even a regex...it's just a static string).  Are you just trying to do prefix matching?